src/evict/evict_lru.c

```diff
@@ -354,6 +354,40 @@ __evict_thread_run(WT_SESSION_IMPL *session, WT_THREAD *thread)
     return (ret);
 }
 
+/*
+ * __clear_saved_walk_tree --
```

Comment:
Please remove the WT-13375 changes from this PR. I'll re-review once it's just the WT-12644 changes